lnrpc/walletrpc/walletkit.proto

```diff
@@ -79,6 +79,28 @@ service WalletKit {
     */
     rpc ListAddresses (ListAddressesRequest) returns (ListAddressesResponse);
 
+    /*
+    SignMessageWithAddr returns a signature with the private key of the provided
+    address. The address lookup in the wallet will consider the default account
+    but can be changed with a cmd argument. In addition the address type is
```

Comment:
> can be changed with a cmd argument

Is this cmd argument supplied somewhere? (same for verifymsg)

Reply:
Correct, old comments from the prior implementation. Now every address not only the default wallet is supported, as long as the wallet is in possession of the private key.